lighthouse-core/gather/driver.js

```diff
@@ -484,6 +485,28 @@ class Driver {
     const networkStatusMonitor = this._networkStatusMonitor;
 
     const promise = new Promise((resolve, reject) => {
+      const logStatus = () => {
```

Comment:
can you add a comment to explain what this is for?
the name doesnt totally help, but i agree the name should remain this short. :)

Reply:
done :)